7xbot.py

```
@@ -60,6 +61,16 @@ def days_until_christmas():
   delta = christmas - today
   return delta.days
 
+  try:
+    load = psutil.Process().cpu_percent(interval=0.1)
+  except Exception:
+    load = psutil.cpu_percent(interval=0.1)
+  """Return basic bot info such as ping, shard ID and CPU load."""
+  ping_ms = round(bot.latency * 1000)
+  shard_id = ctx.guild.shard_id if ctx and ctx.guild else 0
```

Ruff (0.11.9)

Undefined name `ctx`

(F821)